languagetool-core/src/main/java/org/languagetool/rules/neuralnetwork/NeuralNetworkRuleCreator.java

```diff
@@ -0,0 +1,38 @@
+package org.languagetool.rules.neuralnetwork;
+
+import org.languagetool.Language;
+import org.languagetool.rules.Rule;
+import org.languagetool.rules.ScoredConfusionSet;
+import org.languagetool.rules.ScoredConfusionSetLoader;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ResourceBundle;
+
+public abstract class NeuralNetworkRuleCreator {
+  private NeuralNetworkRuleCreator() {}
+
+  private static final String CONFUSION_SET_FILENAME = "neuralnetwork/confusion_sets.txt";
+
+  public static List<Rule> createRules(ResourceBundle messages, Language language, Word2VecModel word2vecModel) {
+    List<ScoredConfusionSet> confusionSets;
+    try(InputStream confusionSetsStream = new FileInputStream(word2vecModel.getPath() + File.separator + CONFUSION_SET_FILENAME)) {
+      confusionSets = ScoredConfusionSetLoader.loadConfusionSet(confusionSetsStream);
+    } catch (IOException e) {
+      System.err.println("Error: " + CONFUSION_SET_FILENAME + " not found for " + language.getShortCode() + ".");
```

Comment:
Maybe just throw an exception?

Reply:
I think that throwing an exception (which is not caught anywhere else) implies that a user always needs all language models if they use any language model. E. g. I could not use word2vec for German only, and don't use it for English, because there would be an uncaught exception when loading the rules for English.

Reply:
I see. In that case, I suggest you test for the existence of the file and print a warning if it's not found. If it is found, the IOException should be thrown (as IOException or in a RuntimeException), as then there's another reason for it than "file not found".